tvb_build/docker/Dockerfile-run

```diff
@@ -39,9 +39,10 @@ RUN wget https://zenodo.org/record/3688773/files/tvb-data.zip?download=1 -O tvb_
     cd tvb_data; \
     /opt/conda/envs/tvb-run/bin/python setup.py develop
 
+ARG lelele
 # make sure a copy of tvb-root exists inside, for users without Github clone
 USER $TVB_USER
-RUN git clone --depth 1 https://github.com/the-virtual-brain/tvb-root.git
+RUN git clone --depth 1 --branch tvb-hpc-rebase https://github.com/the-virtual-brain/tvb-root.git
```

Comment:
This branch switch might not need to end up in master...

Reply:
Nope, it is just for DEV builds.